.github/actions/javascript/reviewerChecklist/index.js

```diff
@@ -30,10 +31,14 @@ function getNumberOfItemsFromReviewerChecklist() {
             });
             res.on('end', () => {
                 const numberOfChecklistItems = (fileContents.match(/- \[ \]/g) || []).length;
+                console.log(`There are ${numberOfChecklistItems} items in the reviewer checklist.`);
                 resolve(numberOfChecklistItems);
             });
         })
-            .on('error', reject);
+            .on('error', (err) => {
+                console.error(err);
+                reject();
```

Comment:
```suggestion
                reject(err);
```
Need to keep passing the error down